js-intro.js

```diff
@@ -5,13 +5,31 @@
 // Consider this variable:
 
 var mantra = "Be the dev"
+var arr = mantra.charAt(0)
 
 // 1a. Write the code that returns 'B' from mantra.
-
+console.log(arr)
 // 1b. Write the code that determines if there is an 'x' in mantra.
-
+function let(){
+    for(let i = 0; i < mantra.length; i++)
+    if(mantra.includes("x")){
+        return "true"
+    }else{
+        return "false"
+    }
+}
+console.log(let(mantra))
```

Comment:
If you are creating a function, make sure you pass arguments into the function and not use the variable itself.